planck-cljs/src/planck/http.cljs

```diff
@@ -249,9 +256,10 @@
   "Performs a POST request. It takes an URL and an optional map of options
```

Comment:
Perhaps better dealt with elsewhere but should it be 'a URL'?

Reply:
Yeah, I suppose your comment is correct and I agree that it is really not related to this particular change.